rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -225,53 +202,60 @@ private static Stream<Arguments> logAddSignatureArgProvider() {
         // Arrange
-        BridgeMainNetConstants bridgeMainNetConstants = BridgeMainNetConstants.getInstance();
-        BlockTxSignatureCache signatureCache = new BlockTxSignatureCache(new ReceivedTxSignatureCache());
-        LinkedList<LogInfo> eventLogs = new LinkedList<>();
-        BridgeEventLogger eventLogger = new BridgeEventLoggerImpl(bridgeMainNetConstants, activations, eventLogs, signatureCache);
-
+        BridgeEventLogger bridgeEventLogger = new BridgeEventLoggerImpl(
+            BRIDGE_CONSTANTS,
+            activations,
+            eventLogs,
+            signatureCache
+        );
         BtcECKey federatorBtcPubKey = federationMember.getBtcPublicKey();
 
-        Keccak256 rskTxHash = PegTestUtils.createHash3(1);
-
-        BtcTransaction btcTxMock = mock(BtcTransaction.class);
-        when(btcTxMock.getHash()).thenReturn(BitcoinTestUtils.createHash(1));
-        when(btcTxMock.getHashAsString()).thenReturn(rskTxHash.toHexString());
-
         // Act
-        eventLogger.logAddSignature(federationMember, btcTxMock, rskTxHash.getBytes());
+        bridgeEventLogger.logAddSignature(federationMember, BTC_TRANSACTION, RSK_TX_HASH.getBytes());
 
         // Assert
         commonAssertLogs(eventLogs);
         assertTopics(3, eventLogs);
 
         CallTransaction.Function bridgeEvent = BridgeEvents.ADD_SIGNATURE.getEvent();
-        Object[] eventTopics = new Object[]{rskTxHash.getBytes(), expectedRskAddress};
+        Object[] eventTopics = new Object[]{RSK_TX_HASH.getBytes(), expectedRskAddress.toString()};
         Object[] eventParams = new Object[]{federatorBtcPubKey.getPubKey()};
 
         assertEvent(eventLogs, 0, bridgeEvent, eventTopics, eventParams);
     }
 
     @Test
     void logReleaseBtc() {
-        // Setup Btc transaction
-        Keccak256 rskTxHash = PegTestUtils.createHash3(1);
-
         // Act
-        eventLogger.logReleaseBtc(btcTx, rskTxHash.getBytes());
+        eventLogger.logReleaseBtc(BTC_TRANSACTION, RSK_TX_HASH.getBytes());
 
         commonAssertLogs(eventLogs);
         assertTopics(2, eventLogs);
-        assertEvent(eventLogs, 0, BridgeEvents.RELEASE_BTC.getEvent(), new Object[]{rskTxHash.getBytes()}, new Object[]{btcTx.bitcoinSerialize()});
+        assertEvent(
+            eventLogs,
+            0,
+            BridgeEvents.RELEASE_BTC.getEvent(),
+            new Object[]{RSK_TX_HASH.getBytes()},
+            new Object[]{BTC_TRANSACTION.bitcoinSerialize()}
+        );
     }
 
     @ParameterizedTest
     @ValueSource(booleans = {false, true})
     void logCommitFederation(boolean isRSKIP383Active) {
-        when(activations.isActive(ConsensusRule.RSKIP383)).thenReturn(isRSKIP383Active);
+        ActivationConfig.ForBlock hopActivations = ActivationConfigsForTest.hop400().forBlock(0);
+        ActivationConfig.ForBlock lovellActivations = ActivationConfigsForTest.lovell700().forBlock(0L);
```

Comment:
shouldn't we use use all activations here?

Reply:
Done